pontos/github/api/tags.py

```diff
@@ -154,10 +154,39 @@ async def get(self, repo: str, tag_sha: str) -> Tag:
                 from pontos.github.api import GitHubAsyncRESTApi
 
                 async with GitHubAsyncRESTApi(token) as api:
-                    tag = await api.tags.get("foo/bar", e746420)
+                    tag = await api.tags.get("foo/bar", "e746420")
                     print(tag)
         """
         api = f"/repos/{repo}/git/tags/{tag_sha}"
         response = await self._client.get(api)
         response.raise_for_status()
         return Tag.from_dict(response.json())
+
+    async def get_tags(self, repo: str) -> AsyncIterator[RepositoryTag]:
```

Comment:
Should be get_all I suppose?
```suggestion
    async def get_all(self, repo: str) -> AsyncIterator[RepositoryTag]:
```